src/TBuffer.h

```diff
@@ -143,7 +149,9 @@ class TBuffer
     TBuffer cut(QPoint&, QPoint&);
     void paste(QPoint&, TBuffer);
     void setBufferSize(int s, int batch);
-    static const QList<QString> getHardCodedEncodingTableKeys() {return csmEncodingTable.keys();}
+    static const QList<QString> getComputerEncodingNames() { return csmEncodingTable.keys(); };
+    static const QList<QString> getFriendlyEncodingNames();
+    static const QString& getComputerEncoding(const QString& encoding);
```

Comment:
Are these `static const` though - `const` is normally ineffective on returns from methods/functions unless it is a _reference_ that is being returned (otherwise it is a value or pointer and `const`entness cannot be enforced or is irrelevant) IIRC.